tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+
+@pytest.fixture(scope='session')
+def capabilities(capabilities):
+    capabilities.setdefault('tags', []).append('socorro')
+    return capabilities
+
+
+@pytest.fixture
+def existing_user(variables):
+    return variables['users']['default']
+
+
+@pytest.fixture
+def selenium(selenium):
+    selenium.implicitly_wait(10)
+    selenium.maximize_window()
```

Comment:
I added the `maximize_window()` back. I find it helps with debugging, particularly when watching playback videos on Sauce.